### PR TITLE
Integrate bidirectional lookup table into frgb

### PR DESCRIPTION
**Pull request by joyhughes**
Complete gamma_LUT class. 
Add static instance to frgb.cpp
Use look up table in FRGB member functions

### CODE REVIEW

File: Lux/frgb.hpp [hunk 1]
```diff
@@ -34,43 +37,58 @@ class frgb {
...
+    inline void setgc( unsigned char g );
+    inline void setbc( unsigned char b );
     inline void setc( unsigned char r, unsigned char g, unsigned char b );
 
-    inline void setul( unsigned int in );
-
+    //inline void setul( unsigned int in );
+    
     // arithmetic operators
+    frgb& operator += (const frgb rhs);
```
Comment (rudi-cilibrasi) [inline Lux/frgb.hpp:68]: these should be const references i think

File: Lux/frgb.hpp [hunk 2]
```diff
@@ -34,43 +37,58 @@ class frgb {
...
+    frgb& operator -= (const frgb rhs);
+    frgb& operator *= (const frgb rhs);
+
+    frgb& operator *= (const float rhs);
```
Comment (rudi-cilibrasi) [inline Lux/frgb.hpp:72]: the const here doesn't mean anything with float in a signature
a const reference would be meaningful. or a plain float is fine.

File: Lux/frgb.cpp [hunk 3]
```diff
@@ -28,43 +25,76 @@ frgb::~frgb() {}
 // question - how to overload [] operator for assignment?
```
Comment (rudi-cilibrasi) [inline Lux/frgb.cpp:25]:
```
float& operator[](int i);
const float& operator[](int i) const;
```

**Review by rudi-cilibrasi — APPROVED** [review]

**Review by rudi-cilibrasi — APPROVED** [review]

**Comment by joyhughes** [comment]
Learned a lot making this first C++ class in many years ... thanks Rudi!

**Pull request merged**